.github/workflows/tests.yml

```diff
@@ -73,13 +70,13 @@ jobs:
 
       - name: Capture hash value
         id: hash
-        run: echo '::set-output name=HASH::${{ runner.os }}-npm-${{ matrix.node }}-${{ hashFiles('**/package-lock.json') }}'
+        run: echo '::set-output name=HASH::${{ runner.os }}-npm-node12-${{ hashFiles('**/package-lock.json') }}'
```

Comment:
Oh hmm, I'm not sure I dig this update.  It's a lot more places to touch to bump the version.  Can we revert and just bump the matrix version to 12?

Reply:
ya sure

Reply:
Done!